clusterman/draining/queue.py

```diff
@@ -226,13 +229,14 @@ def delete_warning_messages(self, hosts: Sequence[Host]) -> None:
 
     def process_termination_queue(
         self,
-        mesos_operator_client: Callable[..., Callable[[str], Callable[..., None]]],
+        mesos_operator_client: Optional[Callable[..., Callable[[str], Callable[..., None]]]],
+        kube_operator_client: Optional[KubernetesClusterConnector],
     ) -> None:
         host_to_terminate = self.get_host_to_terminate()
         if host_to_terminate:
             # as for draining if it has a hostname we should down + up around the termination
-            if host_to_terminate.hostname:
-                logger.info(f'Hosts to down+terminate+up: {host_to_terminate}')
+            if host_to_terminate.scheduler in {None, 'mesos'}:
```

Comment:
If the scheduler is None, would we still need to drain it by mesos pattern?

Reply:
Good point, my thought was that since historically (before we had the `cluster_manger` field) we defaulted to mesos pattern. But since now we have a good mix of both, and assuming the cluster_manager is always specified, I can change this to default to terminate immediately.